src/cdk-experimental/menu/menu.md

```diff
@@ -0,0 +1,167 @@
+The `@angular/cdk-experimental/menu` module provides you with a way to easily create complex menu interfaces with support for:
+
+- Standalone Menus
+- Menubars
+- Context Menus
+- Inline Menus
+- MenuItems
+- RadioButtonItems
+- CheckboxItems
```

Comment:
I would break this into a section called "Supported ARIA roles" and use the role names from the spec, combining with some of the text you have below.